.travis.yml

```diff
@@ -29,21 +28,13 @@ script:
   - bundle exec rake
 
 env:
-  - "RAILS_VERSION=3.2.21"
-  - "RAILS_VERSION=4.0.13"
-  - "RAILS_VERSION=4.1.14"
   - "RAILS_VERSION=4.2.8"
   - "RAILS_VERSION=4.2.8 DATABASE_URL=mysql2://root@localhost/statesman_test"
   - "RAILS_VERSION=4.2.8 DATABASE_URL=postgres://postgres@localhost/statesman_test"
-  - "RAILS_VERSION=5.0.0 EXCLUDE_MONGOID=true"
+  - "RAILS_VERSION=5.0.0 EXCLUDE_MONGOID=true DATABASE_URL=mysql2://root@localhost/statesman_test"
```

Comment:
I think we're adding 5.0.0 and 5.1.0 in other PRs. Shall we limit this one to the removal of the old versions?